packages/material/src/complex/MaterialTableControl.tsx

```diff
@@ -154,6 +160,7 @@ interface NonEmptyCellProps extends OwnPropsOfNonEmptyCell {
 interface OwnPropsOfNonEmptyCell {
   rowPath: string;
   propName?: string;
+  title: string;
```

Comment:
We don't need title in normal cells, right?
```suggestion
```

Reply:
done!